src/pudl/etl/__init__.py

```diff
@@ -39,6 +39,9 @@
     *load_assets_from_modules([pudl.transform.ferc1], group_name="norm_ferc1"),
     *load_assets_from_modules([pudl.extract.ferc714], group_name="raw_ferc714"),
     *load_assets_from_modules([pudl.transform.ferc714], group_name="clean_ferc714"),
+    *load_assets_from_modules(
```

Comment:
This shouldn't actually be here - instead, we should add a new extraction module for Census data that is loaded here!